script/mobile-nav-menu.js

```diff
@@ -0,0 +1,20 @@
+/* Variable Declarations */
+const hamburger = document.querySelector('.hamburger');
+const navMenuMobile = document.querySelector('.nav-menu-mobile');
+
+/* Function Declarations */
+function openMenu() {
+  hamburger.classList.toggle('active');
+  navMenuMobile.classList.toggle('active');
+}
+
+function closeMenu() {
+  hamburger.classList.remove('active');
+  navMenuMobile.classList.remove('active');
+}
+
+// Open Nav Menu for touchstart
+hamburger.addEventListener('touchstart', openMenu);
+
+// Close Nav Menu for touchend
+document.querySelectorAll('.nav-link-mobile').forEach((n) => n.addEventListener('touchend', closeMenu));
```

Comment:
- [ ] Great job so far 👏 But I can't view your mobile menu. Make sure to fix it and display it on chrome.

https://user-images.githubusercontent.com/70269507/173341731-b9760bff-d05f-4fc5-91e6-1ec7b961780c.mov

As I was not able to view your mobile menu, I could not check:

- [ ] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [ ] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.
- [ ] The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins).